ACE/include/makeinclude/platform_linux_common.GNU

```diff
@@ -1,15 +1,18 @@
 # -*- Makefile -*-
 
-# We always include config-linux.h on Linux platforms.
+# Include config-linux.h by default on Linux platforms.
 ACE_PLATFORM_CONFIG ?= config-linux.h
 
 debug ?= 1
 optimize ?= 1
 threads ?= 1
 insure ?= 0
+linux_is_cross_compiled ?= 0
```

Comment:
Will this have any impact when using gcc with cross compilation to ARM?

Reply:
I just realized I have the checks for this backwards down below, so I will have to fix that, but no I don't think so. I'm only trying to make it act different for Android at the moment.

Reply:
Actually, after playing around with it a little more, I think my changes to this file and `platform_android.GNU` are unnecessary and I will reverse them.